client/components/DarkModeSwitch.js

```diff
@@ -0,0 +1,20 @@
+import { useColorMode, IconButton } from '@chakra-ui/react'
+import { SunIcon, MoonIcon } from '@chakra-ui/icons'
+
+const DarkModeSwitch = () => {
+    const { colorMode, toggleColorMode } = useColorMode()
+    const iconColor = {
+        light: 'black',
+        dark: 'white'
+    }
+    return (
+        <IconButton
+            aria-label="Toggle dark mode"
+            icon={colorMode === 'dark' ? <SunIcon /> : <MoonIcon />}
+            onClick={toggleColorMode}
+            color={iconColor[colorMode]}
+        />
+    )
+}
```

Comment:
Seems to work well in general 👍 But the options in the multi-select do not show up well in dark mode - we may have to see how to adjust the colors for that in different modes

Reply:
Alright, will look into it!

Reply:
I've tried many ways, such as the `<LightMode>` override but it doesn't seem to work... the issue could be with Chakra's dark mode implementation on the `<Select>` element.

I've referenced an issue on the main Chakra UI repo–seems to be a known issue: [https://github.com/chakra-ui/chakra-ui/issues/417#issue-566611352](https://github.com/chakra-ui/chakra-ui/issues/417#issue-566611352).

I saw the solution posted, but unsure of how to go about the CSS reset. Do you have any idea? @joanneong

Reply:
@theodorayy Fixed as per the solution shared in your link. I created the select style in `DarkModeSwitch.js`, and then used the style for the `Select` element in the `recycle-and-reuse` and `geolocation` pages. Feel free to test it out.

Reply:
You're awesome––thanks for figuring it out.